<launch>
    <arg name="voxel_size" default="0.20"/>
    <arg name="num_agents" default="1"/>
    <arg name="param_file" default="$(find multi_robot_global_planner)/cfg/parameters_vicon.yaml" />
    <arg name="launch_prefix" default=""/>

    <node name="multi_robot_global_planner" pkg="multi_robot_global_planner" type="multi_robot_global_planner_node" output="screen" args="-alsologtostderr" clear_params="true" launch-prefix="$(arg launch_prefix)">
       <rosparam command="load" file="$(arg param_file)" />
       <remap from="multi_robot_global_planner/esdf_map_in" to="esdf_map" />
       <remap from="multi_robot_global_planner/tsdf_map_in" to="tsdf_map" />
    </node>

</launch>
